### PR TITLE
feat(sound): cockpit sounds part 4

### PR DESCRIPTION
**Pull request by ImenesFBW**
## Summary of Changes
Additional changes to cockpit soundscape:

- all round-buttons, like chrono, calls, rudder trim, etc.. (except fire test and the three on the FCU) now have their own custom roundbutton sound event.
- Red sidestick button
- DCDU buttons
- MCDU brightness 
- Fire test buttons on the fire panel now have their own firetestbutton event (the same event is used for the sidestick AP disengage)

## Additional context

Credit for the custom wwise events goes to @hotshotp 

Discord username: Imenes#0001

## Testing instructions

Test all the buttons described above, and please also click all the square push buttons to make sure they don't play the new roundbutton sound but remain the same as they were before.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by hotshotp — APPROVED** [review]
LGTM!

**Comment by NUTTTTTTTT** [comment]
Quality Assurance Tester Report

Discord: █▀█ █▄█ ▀█▀#2123
Object of testing: #PR 5630
Tier of Testing: 1
Date: 13/08/2021

Testing Process:
Loaded in on runway, tested sounds for:
- All round buttons: Chrono, calls, rudder trim, door video etc.
- A/P disconnect sidestick button.
- DCDU buttons + MCDU brightness.
- Fire test buttons - had the same sounding click as A/P disconnect.
- Previous square buttons to confirm were the same.

Negatives: N/A
Testing Results: Passed

Conclusions: Adds a lot of previously missing sounds, and all sound unique and realistic, great stuff! Definitely a great addition and all other buttons sounded the same as when tested in the main development branch just before.

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 1]
```diff
@@ -1682,6 +1682,7 @@
                         <MOMENTARY/>
                         <NO_SEQ1 />
                         <NO_SEQ2 />
+                        <ROUND/>
```
Comment (Benjozork) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:1685]:
```suggestion
                        <ROUND />
```

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 2]
```diff
@@ -1760,6 +1761,7 @@
                             <MOMENTARY/>
                             <NO_SEQ1 />
                             <NO_SEQ2 />
+                            <ROUND/>
```
Comment (Benjozork) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:1764]:
```suggestion
                            <ROUND />
```

**Review by Benjozork — CHANGES_REQUESTED** [review]

**Review by Benjozork — APPROVED** [review]

**Comment by Samueleonard** [comment]
Quality Assurance Tester Report

Discord:: 1samueleonard1
Object of testing: (#5630 )
Tier of Testing: (1)
Date: (13.08.2021)

Testing Process: 
. loaded in c+d
. test all sounds as written above and listened for new sounds

Negatives: as already noted by someone else on discord - ISIS button does not have a 'push' sound but model limitation
Testing Results: (Passed)

Conclusions: lgtm
Media: n/a

**Comment by KemalOues** [comment]
Today I also have no rain sounds in the latest Dev Release

**Comment by hotshotp** [comment]
> Today I also have no rain sounds in the latest Dev Release

Known issue

**Pull request merged**